pages/users/[id].vue

```diff
@@ -103,7 +103,7 @@
 
       <div v-if="user.organizations.length">
         <h2 class="text-sm font-bold uppercase mb-3">
-          {{ $t('{n} organisation | {n} organizations', { n: user.organizations.length }) }}
+          {{ $t('{n} organisations', { n: user.organizations.length }) }}
```

Comment:
We could keep the plural here?

Reply:
It is, `{n} organisations` is already defined and used, with the `"{n} organisations | {n} organisation | {n} organisations"` translation

Reply:
I think we should remove the locales' JSON files, there are useless to maintain.